public/locales/en/aspectRatio.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Any chance we can move all of these translation files to an `i18n` directory in the root of the project?